src/adapters/postgres/user-adapter.ts

```
@@ -1141,6 +1144,7 @@ export class PostgresUserService implements IServicelocator {
         academicYearId,
         response
       );
+      console.log('hi2', result);
```

ESLint

[error] Replace `'hi2'` with `"hi2"`

(prettier/prettier)